tests/unit/utils/test_vector_format.py

```diff
@@ -0,0 +1,83 @@
+import json
+import pathlib
+
+import fiona
+import pytest
+import shapely
+from rasterio.crs import CRS
+from upath import UPath
+
+from rslearn.const import WGS84_PROJECTION
+from rslearn.utils.feature import Feature
+from rslearn.utils.geometry import Projection, STGeometry
+from rslearn.utils.vector_format import GeojsonCoordinateMode, GeojsonVectorFormat
+
+
+@pytest.mark.parametrize(
```

Comment:
Suggestion: Parametrize is ideal imo when you can specify both the inputs and outputs within the same logic. If you need separate logic for each case I would recommend just having separate unit tests so it is more readable.